test/test-cases/multiple-routes/app/sku.config.js

```diff
@@ -5,14 +5,14 @@ export default {
       host: 'dev.seek.com.au',
       routes: [
         { route: '/', name: 'home' },
-        { route: '/details/$id', name: 'details' },
+        { route: '/details/:id', name: 'details' },
```

Comment:
Should we keep these as `$`?

Reply:
Actually, maybe worth keeping one as `$` and the other as `:` so this config tests both at once.